frontend/jest.config.ts

```diff
@@ -0,0 +1,28 @@
+import type { Config } from '@jest/types';
```

Comment:
As a follow up to your PR here - I really would like to de-dupe the jest config from hac-dev, hac-core, hac-bs, etc. Maybe we can do that through the SDK. At least that way our configs will be consistent and not vary across repos. 

https://github.com/openshift/dynamic-plugin-sdk/blob/main/jest.config.ts
https://github.com/openshift/hac-dev/blob/main/jest.config.js
https://github.com/RedHatInsights/frontend-components/blob/master/jest.config.js
https://github.com/openshift/console/blob/master/frontend/packages/eslint-plugin-console/lib/config/jest.js